site/gatsby-site/migrations/2023.08.17T15.56.33.fix-report-users.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config');
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  const reportsCollection = client.db(config.realm.production_db.db_name).collection('reports');
+
+  const updateResult = await reportsCollection.updateMany(
+    {
+      user: undefined,
+    },
+    { $unset: { user: '' } }
+  );
+
+  console.log('Update result:', updateResult);
+};
```

Comment:
Please consider adding the empty `down` function even though there's no logical way to revert -- this way the node migrator tool won't throw an error on down migrations.

Reply:
added